simpleclient_common/src/main/java/io/prometheus/client/exporter/common/TextFormat.java

```diff
@@ -11,6 +11,7 @@
    * Content-type for text version 0.0.4.
    */
   public final static String CONTENT_TYPE_004 = "text/plain; version=0.0.4; charset=utf-8";
+  public final static String CONTENT_TYPE = "text/plain";
```

Comment:
This doesn't belong here, this is not the content type of the Prometheus text format